theme/typography.ts

```diff
@@ -0,0 +1,59 @@
+import { ColorStates } from "./constants";
+
+interface Measurements {
+  [key: string]: string;
+}
+
+interface TextPalette extends ColorStates {
+  primary: string;
+  secondary: string;
+  disabled: string;
+  pressed: string;
+}
+
+interface Typography {
+  palette: TextPalette;
+  lineHeights: Measurements;
+  sizes: Measurements;
+}
+
+const FONT_SIZES = [12, 14, 16, 20, 24, 30, 36, 48, 60];
+const PRIMARY_TEXT_COLOR = "";
+const SECODARY_TEXT_COLOR = "";
+const PRESSED_TEXT_COLOR = "";
+const DISABLED_TEXT_COLOR = "";
+
+const palette: TextPalette = {
+  primary: PRIMARY_TEXT_COLOR,
+  secondary: SECODARY_TEXT_COLOR,
+  disabled: DISABLED_TEXT_COLOR,
+  pressed: PRESSED_TEXT_COLOR
+};
+
+const lineHeights: Measurements = {
+  xs: "19px",
+  sm: "23px",
+  rg: "26px",
+  md: "32px",
+  lg: "39px",
+  xl: "49px"
+};
+const sizes: Measurements = {
```

Comment:
nit: space

Reply:
Space? Remove space?

Reply:
![image](https://user-images.githubusercontent.com/1918027/56514160-8eb3d180-6502-11e9-91c9-e131d77405f9.png)

Reply:
lol I was referring to add a new line before the `const`

Reply:
got it.